packages/graphback-runtime/src/data/GraphbackDataProvider.ts

```diff
@@ -54,6 +54,15 @@ export interface GraphbackDataProvider<Type = any, GraphbackContext = any> {
      */
     findAll(context?: GraphbackContext): Promise<Type[]>;
 
+    /**
+     * Implementation for finding all objects with support for pagination
+     * 
+     * @param limit Limit on number of objects to return 
+     * @param offset Number of objects to exclude starting from the first
+     * @param context context object passed from graphql or rest layer 
+     */
+    findMore(offset: number, limit: number, context?: GraphbackContext): Promise<Type[]>
```

Comment:
Let's remove this method and use find All with new type:

`GraphbackPage: {limit, offset}`

Reply:
On it

Reply:
Where do you think I should put the GraphbackPage type?